popup-app/components/NoParent.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const NoParent = ({ appName }) => (
+  <div>
+    This window was opened to log you in to{' '}
+    <span className="app-name">{appName}</span>, but that app is no longer open.
+    You can close this window.
```

Comment:
Instead of "you can close", maybe some advice for success, such as "Please close this window and go back to the original app to log in again."?

Reply:
Good point!